src/widget/wcoverartlabel.h

```diff
@@ -33,9 +33,7 @@ class WCoverArtLabel : public QLabel {
   private:
     QPixmap m_loadedCover;
     TrackPointer m_pLoadedTrack;
-    WCoverArtMenu* m_pCoverMenu;
-    DlgCoverArtFullSize* m_pDlgFullSize;
+    QPointer<WCoverArtMenu> m_pCoverMenu;
+    QPointer<DlgCoverArtFullSize> m_pDlgFullSize;
```

Comment:
Why QPointer?

Reply:
Because this is how you safely store pointers to objects whose lifetime is managed by the Qt runtime.

Reply:
No, the -> operator will segfault like before.
QPointer documents just the opposite of patented pointer: the tracked QObject is not owned by "this".